shared/domains/chelonia/db.js

```diff
@@ -149,6 +149,10 @@ export default (sbp('sbp/selectors/register', {
       await sbp('chelonia/db/set', entry.hash(), entry.serialize())
       await sbp('chelonia/db/set', getLogHead(contractID), JSON.stringify({ HEAD: entry.hash(), height: entry.height() }))
       console.debug(`[chelonia.db] HEAD for ${contractID} updated to:`, entry.hash())
+      if (entryPreviousHEAD) {
+        await sbp('chelonia/db/set', `next=${entryPreviousHEAD}`, entry.hash())
+        console.debug(`[chelonia.db] next hash for ${entryPreviousHEAD} updated to:`, entry.hash())
```

Comment:
I think we have enough logging for this function, and adding this extra logging would be a bit unnecessary and excessive. Can remove?

Reply:
Good catch